sql/migrations/20210810191400_world.sql

```diff
@@ -590,6 +590,9 @@ UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = -50 WHERE `item` = 3
 -- Correct Diplay ID For Energetic Rod
 UPDATE `item_template` SET `display_id` = 30915 WHERE `entry` = 18321;
 
+-- Remove Teebus Blazing Longsword From Lockboxes
+DELETE FROM `reference_loot_template` WHERE `entry` = 1014 AND `item`=1728;

```

Comment:
It could drop from Heavy Junkbox according to thottbot snapshot from 2006.
https://web.archive.org/web/20090313074825/http://thottbot.com:80/i16885#D::A

Reply:
Oh no, wait. That's 2009, somehow I read the last digit upside down...

Reply:
I think it was added sometime in tbc to lockboxes

Reply:
Ok